lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -384,6 +387,9 @@ states:
       enhanced-verification:
         targetJourney: F2F_HAND_OFF
         targetState: START
+      fraud-fail-with-no-ci:
+        targetJourney: FAILED
+        targetState: FAILED
```

Comment:
These other states for P1 are more interesting... They won't be valid for M1C, but could be for L1B.

Feels fine to leave this as failed for now, and we can update if/when we support L1B.